packages/barqode/src/lib/components/barqode-dropzone.svelte

```diff
@@ -8,11 +8,11 @@
 		onDragover,
 		onError,
 		children,
+		inputRef = $bindable(null),
+		wrapperRef = $bindable(null),
```

Comment:
Exposed here @ollema

Reply:
no I meant the other component. you have defined it in both types I think

Reply:
Oh yeah! Done ✅